src/busy-indicator.scss

```diff
@@ -79,6 +88,16 @@ $fd-dots-size-l: 2rem;
   height: $fd-dots-size-l;
 }
 
+.#{$block}--label {
+  display: block;
+  margin: auto;
+  font-size: var(--sapFontSize);
+  font-family: var(--sapFontFamily);
+  font-weight: normal;
```

Comment:
These things are coming from the reset function that you are missing

Reply:
fixed